.github/workflows/testing.yml

```diff
@@ -25,9 +25,8 @@ jobs:
         tarantool:
           - '1.10'
           - '2.8'
-          - '2.x-latest'
+          - '2.10'
```

Comment:
Added step to install latest Tarantool 2 nightly.

Reply:
From the installation log:
```
Warning: Unexpected input(s) 'tarantool-nightly', valid inputs are ['tarantool-version', 'nightly-build', 'cache-key']
```
https://github.com/tarantool/tarantool-python/runs/7407841452?check_suite_focus=true

Reply:
Yeah, sorry, I'll go and fix that one

Reply:
Well, it seems like there is no nightly for series-2 yet. And there is no reason to run tests with pre-release 2.10 if release is newer. Since there is a reusable_testing.yml pipeline which should be triggered by tarantool/tarantool repo master, I think we may ignore this for now.

Reply:
I would note that it is possible that we'll want to offer a release candidate between releases. I'm thinking on this here: https://github.com/tarantool/setup-tarantool/issues/23. Join if you have an idea regarding a good API.